.rubocop_todo.yml

```diff
@@ -1,16 +1,24 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-03-27 08:53:54 -0700 using RuboCop version 0.73.0.
+# on 2020-03-31 15:59:53 -0700 using RuboCop version 0.73.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: squiggly, active_support, powerpack, unindent
+Layout/IndentHeredoc:
```

Comment:
Since this cop claims to support auto-correct, are you open to running auto-correct and then re-generating the TODO config?